lua/autorun/server/sv_cfc_vote_init.lua

```diff
@@ -0,0 +1,18 @@
+local function addFiles( dir )
+    local files, dirs = file.Find( dir .. "/*", "LUA" )
+    if not files then return end
+    for k, v in pairs( files ) do
+        if string.match( v, "^.+%.lua$" ) then
+            AddCSLuaFile( dir .. "/" .. v )
+        end
+    end
+    for k, v in pairs( dirs ) do
+        addFiles( dir .. "/" .. v )
+    end
+end
```

Comment:
a loading system like this is a bit unnecessary when you only have a couple of files
just adding all the files explicitly might be better